api/v1alpha1/bundle_types.go

```diff
@@ -50,6 +50,10 @@ type BundleSpec struct {
 	ProvisionerClassName string `json:"provisionerClassName"`
 	// Source defines the configuration for the underlying Bundle content.
 	Source BundleSource `json:"source"`
+	// Secret contains the secret name that has authorization information for the source and is in the namespace that the provisioner is deployed.
+	// for image source, the secret contains the image pull secret.
+	// for git source, the secret contains the user ID (key: user) and the personal access token to the repository (key: token)
```

Comment:
I wonder if we want to support auth via ssh as well? I think we can take inspiration from source-controller's implementation https://github.com/fluxcd/source-controller/blob/065a7607523dde326c3d513461befb047bd77eff/api/v1beta2/gitrepository_types.go#L62

Reply:
Yes, we do.  Can it be a separate issue?